target/linux/ramips/dts/OMEGA2P.dts

```diff
@@ -0,0 +1,22 @@
+/dts-v1/;
+
+#include "OMEGA2.dtsi"
+
+/ {
+	model = "Onion Omega2+";
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x8000000>;
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		system {
+			label = "omega2p:amber:system";
+			gpios = <&gpio1 12 GPIO_ACTIVE_LOW>;
+			default-state = "on";
```

Comment:
You missed to answer my question regarding the default state on, I asked the last time:

>Have you added the default state to enable the system LED as early as possible, to keep the LED switched on (is >already switched on by bootloader) or for another reason?